File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0108U00InitGrdMasterListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0108U00InitGrdMasterListItemInfo {
	private String codeType;
	private String codeTypeName;

	public CPL0108U00InitGrdMasterListItemInfo(String codeType,
			String codeTypeName) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
	}

	public String getCodeType() {
		return codeType;
	}

	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}

	public String getCodeTypeName() {
		return codeTypeName;
	}

	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0108U00InitGrdMasterListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
    public class CPL0108U00InitGrdMasterListItemInfo
    {
        private String _codeType;
        private String _codeTypeName;

        public String CodeType
        {
            get { return this._codeType; }
            set { this._codeType = value; }
        }

        public String CodeTypeName
        {
            get { return this._codeTypeName; }
            set { this._codeTypeName = value; }
        }

        public CPL0108U00InitGrdMasterListItemInfo() { }

        public CPL0108U00InitGrdMasterListItemInfo(String codeType, String codeTypeName)
        {
            this._codeType = codeType;
            this._codeTypeName = codeTypeName;
        }

    }
}